server/auth/auth_test.go

```diff
@@ -721,6 +747,15 @@ func TestConnect(t *testing.T) {
 	}) // 1:1 = 50%
 	defer rig.storage.setRatioData(&ratioData{}) // clean slate
 
+	// TODO: update tests now that there is are close/ban and unban operations,
+	// instead an integral tier.
+
+	// TODO: update tests now that cancel ratio is part of the score equation
+	// rather than a hard close operation. But the cancel ratio is stupid and
+	// generally disabled in production anyway so maybe screw it.
+
+	/* cancel ratio stuff we should probably just remove
```

Comment:
Dang I forgot we decided not remove this after all.  I'll work on these tests in #1820